docs/user_guide/mitigation.rst

```diff
@@ -80,11 +81,408 @@ Reductions
+
+    >>> ClassificationMoment(difference_bound=0.01)  # doctest: +SKIP
+
+Note that achieving parity with the :code:`difference_bound` does not mean
+that the difference between the group with the highest metric value and the
+group with the lowest metric value is below the :code:`difference_bound`, but
+rather the difference between each individual group and the metric value for
+the overall population (or cohort thereof) is below :code:`difference_bound`. 
+
+In constrast, ratio-based constraints consider the ratio as opposed to the
+difference. Parity is represented as
+
+.. math::
+
+    r \leq \dfrac{\text{metric}_a}{\text{metric}_*} \leq \dfrac{1}{r} \; \forall a
+
+with a :code:`ratio_bound` :math:`r`. Similar to the :code:`difference_bound`
+for difference-based constraints, ratio-based constraints have a
+:code:`ratio_bound_slack`. The two inequalities can also be considered
+separately as
+
+.. math::
+
+   r \leq \dfrac{\text{metric}_a}{\text{metric}_*} \; \forall a \\
+   \dfrac{\text{metric}_a}{\text{metric}_*} \leq \dfrac{1}{r} \; \forall a
+
+and one step further in the equivalent form:
+
+.. math::
+
+   - \text{metric}_a + r \cdot \text{metric}_* \leq 0 \; \forall a \\
+   r \cdot \text{metric}_a - \text{metric}_* \leq 0 \; \forall a
+
+To use a constraint in its ratio-based version simply indicate as much through
+its arguments
+
+    >>> ClassificationMoment(ratio_bound=0.9, ratio_bound_slack=0.01)
+
+In other words, this means that
+:math:`0.9 \cdot \text{metric}_a \leq \text{metric}_*`, i.e., the metric value for
+each group needs to be at least 90% of the overall metric value, and
+:math:`0.9 \cdot \text{metric}_* \leq \text{metric}_a`, i.e., the overall metric
+value needs to be at least 90% of each group's metric value.
+
+.. note::
+
+    It is not possible to specify both :code:`difference_bound` *and*
+    :code:`ratio_bound` for the same constraint object.
+
+.. _demographic_parity:
+
+Demographic Parity
+^^^^^^^^^^^^^^^^^^
+
+A classifier :math:`h(X)` satisfies *Demographic Parity* if
+
+.. math::
+    
+    \P[h(X) = y' \given A = a] = \P[h(X) = y'] \; \forall a, y'
+ 
+In other words, the selection rate or percentage of samples with label 1
+should be equal between all groups. Implicitly this means the percentage
+with label 0 is equal as well.
+
+In the example below class :code:`"a"` has a selection rate of 60%,
+:code:`"b"` has a selection rate of 20%. The overall selection rate is 40%,
+so :code:`"a"` is `0.2` above the overall selection rate, and :code:`"b"` is
+`0.2` below. The seemingly redundantly stored constraint violation with both
+:code:`"a"` and :code:`"b"` showing up twice stems from the constraint
+definition outlined in :ref:`constraints_binary_classification`. Each equality
+constraint is written as two inequalities of which one contains
+:math:`\text{metric}_a` positively (sign :math:`+`) and one negatively
+(sign :math:`-`).
+Note that the specified :code:`difference_bound` is not used
```

Comment:
I don't think this sentence makes a lot of sense.

Reply:
Why not? The difference bound is indeed not used because we're not doing any mitigation here. The fact that *a* difference bound was provided is used to calculate gamma, but not the difference bound itself.

Reply:
The point is, if it's unused, why talk about it?